public/FluentUI.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#if os(iOS)
+@_exported import FluentUI_ios
+#elseif os(macOS)
+@_exported import FluentUI_macos
+ #endif
```

Comment:
is there a weird indentation here?

Reply:
Yes, a pesky space snuck in. Good eye! 👀